test/dlc_tests/sanity/test_anaconda.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from invoke.context import Context
+from test import test_utils
+
+
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.model("N/A")
+@pytest.mark.integration("anaconda_removal")
+def test_repo_anaconda_not_present(image):
+    """ Test to see if all packages installed in the image do not come from repo.anaconda.com """
+    ctx = Context()
+    container_name = test_utils.get_container_name("anaconda", image)
+    test_utils.start_container(container_name, image, ctx)
+
+    # First check to see if image has conda installed, if not, skip test since no packages installed from conda present
+    conda_present = test_utils.run_cmd_on_container(container_name, ctx, "find . -name conda").stdout.strip()
+    if not conda_present:
+        pytest.skip("Image does not have conda installed, skipping test.")
```

Comment:
nit: helpful to include which image is being tested, somethignlike

```
pytest.skip(f"Image {image} does not have conda installed, skipping test.")
```

In this case you'll want to stop and remove the container, esp since we're not even using it to test anything

Reply:
updated